Sources/CodableKit/Reflection/ReflectionDecodable.swift

```diff
@@ -94,42 +99,74 @@ extension ReflectionDecodable where Self: Equatable {
 extension String: ReflectionDecodable {
     /// See `ReflectionDecodable`.
     public static func reflectDecoded() -> (String, String) { return ("0", "1") }
+
+    public static var isPrimitiveType: Bool { return true }
 }
 
 extension FixedWidthInteger {
     /// See `ReflectionDecodable`.
     public static func reflectDecoded() -> (Self, Self) { return (0, 1) }
 }
 
-extension UInt: ReflectionDecodable { }
-extension UInt8: ReflectionDecodable { }
-extension UInt16: ReflectionDecodable { }
-extension UInt32: ReflectionDecodable { }
-extension UInt64: ReflectionDecodable { }
+extension UInt: ReflectionDecodable {
+    public static var isPrimitiveType: Bool { return true }
```

Comment:
All public methods need doc blocks. In this case, you can just do something like:

```swift
/// `ReflectionDecodable` conformance.
```